Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -158,6 +158,26 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
         .AddParameter("expression", _("Bottom Y position"))
         .AddParameter("expression", _("Chamfer (in pixels)"))
         .SetFunctionName("DrawChamferRectangle");
+  
+  obj.AddAction("RegularPolygon",
+                _("Regular Polygon"),
+                _("Draw a regular polygon on screen"),
+                _("Draw at _PARAM1_;_PARAM2_ a regular polygon with _PARAM4_ sides and radius: "
+                  "_PARAM3_ (rotation: _PARAM5_) "
+                  "with _PARAM0_"),
+                  _("Drawing"),
+                  "res/actions/regularPolygon24.png",
+                  "res/actions/regularPolygon.png")
+
+        .AddParameter("object", _("Shape Painter object"), "Drawer")
+        .AddParameter("expression", _("X position of center"))
+        .AddParameter("expression", _("Y position of center"))
+        .AddParameter("expression", _("Radius (in pixels)"))
+        .AddParameter("expression",
+              _("Number of sides of the polygon (minimum: 3)"))
```

Comment:
I reckon you use the same order as for DrawStar so that when one change from one to the other, they have fewer parameters to change (number of sides would correspond to number of points of the star)